### DISCUSSION TITLE
Public Repository Owner

### DISCUSSION THREAD

**Discussion by reemasharma19** [community/community#125364] [New to GitHub] [answered]
Where I can find the owner of the public repository? I only see folks as contributor but no owner details visible.

**Comment by davevad93** [comment, marked as answer]
Hi @reemasharma19 ,

At the top left corner you can see the name of the repo owner right next to the repo name.
In the following screenshot sample `torvalds` is the owner of the `linux` repo:

![1](https://github.com/community/community/assets/94308234/da059f86-e145-4223-a5cd-1c80f31e1819)
